ConwaysGameOfLife.Tests/UniverseShould.cs

```diff
@@ -37,7 +37,7 @@ public void WhenUniverseGenerated_ReturnsListOfNeighbours()
 
             var result = universe.GetCellNeighbourLocations(cell);
 
-            Assert.Equal(expected, result);
+            Assert.True(UniverseHelper.ListsOfLocationsAreEqual(expected, result));
             Assert.Equal(8, result.Count);

```

Comment:
the count equality should be encapsulated in the first assertion
```suggestion
            Assert.True(UniverseHelper.ListsOfLocationsAreEqual(expected, result));
```